packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -192,6 +192,7 @@ export function computeDefaults<T = any, S extends StrictRJSFSchema = RJSFSchema
         experimental_defaultFormStateBehavior,
         parentDefaults: Array.isArray(parentDefaults) ? parentDefaults[idx] : undefined,
         rawFormData: formData as T,
+        required,
```

Comment:
Is this a meaningful enough set of changes to mention in the changelog?

Reply:
Whoops! This should not be here yet